kedro-training/docs/04_new_project.md

```diff
@@ -172,3 +172,6 @@ cars:
   bucket_name: my_bucket
```

Comment:
I may be wrong but doesn't this have to be updated to this: https://kedro.readthedocs.io/en/stable/04_user_guide/04_data_catalog.html#using-the-data-catalog-with-the-yaml-api

Reply:
I'm addressing this in https://github.com/quantumblacklabs/kedro-examples/pull/35 :)